.rubocop.yml

```diff
@@ -2,7 +2,6 @@ inherit_gem:
   rubocop-shopify: rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.7
```

Comment:
I think we need to bump Rubocop to 1.61.0 in order for this to work https://github.com/Shopify/erb-lint/blob/main/Gemfile.lock#L78